typings/balena-pine.d.ts

```diff
@@ -14,3 +14,14 @@ export interface Pine {
 	): Bluebird<'OK'>;
 	upsert<T>(params: PineClient.UpsertParams<T>): Bluebird<T | 'OK'>;
 }
+
+export type PineWithSelectOnGet = Omit<Pine, 'get'> & {
```

Comment:
I initially tried having just a `WithSelectOnGet<T>` type helper using mapped types, but currently mapped typed in TypeScript only pick the last (or maybe most generic) overload of the method.